src/dashboard/Settings/actions.js

```diff
@@ -0,0 +1,53 @@
+/* Actions for the incident */ /* const actions */
```

Comment:
Please read [contribution guide](https://github.com/CodeTanzania/emis-dashboard/blob/develop/CONTRIBUTING.md) on how to write actions and action types